### PR TITLE
php-cs-fixer: apply heredoc_indentation rule.

### PR DESCRIPTION
**Pull request by demiankatz**
This applies one of the rules that will be added in https://github.com/vufind-org/vufind/pull/2795, to reduce diffs over there.

### CODE REVIEW

File: themes/bootstrap3/templates/Recommend/TopFacets.phtml [hunk 1]
```diff
@@ -53,9 +53,9 @@
 <?php endif; ?>
 <?php
 $script = <<<JS
-VuFind.truncate.initTruncate('.top-facets-contents');
-// Close with JS (ensures content is shown to non-JS users)
-$(this).find('.collapse').removeClass('in');
-JS;
+                      VuFind.truncate.initTruncate('.top-facets-contents');
```
Comment (EreMaijala) [inline themes/bootstrap3/templates/Recommend/TopFacets.phtml]: This one seems a bit odd. Any idea what's going on here?

File: themes/bootstrap3/templates/Recommend/VisualFacets.phtml [hunk 2]
```diff
@@ -45,12 +45,12 @@
     );
 
     $script = <<<JS
-  VuFind.addTranslations($translations);
-  var pivotdata = $pivotData;
-  jQuery(document).ready(function(data) {
-      showVisualFacets(pivotdata);
-});
-JS;
+          VuFind.addTranslations($translations);
```
Comment (EreMaijala) [inline themes/bootstrap3/templates/Recommend/VisualFacets.phtml]: Two spaces too far?

File: themes/bootstrap3/templates/record/hold.phtml [hunk 3]
```diff
@@ -142,14 +142,14 @@
...
     $js = <<<JS
-        if ($.isReady) {
-            setUpHoldRequestForm("{$this->escapeHtml($this->driver->getUniqueId())}");
-        } else {
-            $(document).ready(function() {
-                setUpHoldRequestForm("{$this->escapeHtml($this->driver->getUniqueId())}");
-            });
-        }
-JS;
+                if ($.isReady) {
```
Comment (EreMaijala) [inline themes/bootstrap3/templates/record/hold.phtml]: Too much indentation?

File: themes/bootstrap3/templates/record/illrequest.phtml [hunk 4]
```diff
@@ -115,14 +115,14 @@
...
     $js = <<<JS
-        if ($.isReady) {
-            setUpILLRequestForm("{$this->escapeHtml($this->driver->getUniqueId())}");
-        } else {
-            $(document).ready(function() {
-                setUpILLRequestForm("{$this->escapeHtml($this->driver->getUniqueId())}");
-            });
-        }
-JS;
+                if ($.isReady) {
```
Comment (EreMaijala) [inline themes/bootstrap3/templates/record/illrequest.phtml]: Too much indentation?

File: themes/bootstrap3/templates/record/storageretrievalrequest.phtml [hunk 5]
```diff
@@ -100,16 +100,16 @@
...
-    if ($('#storageRetrievalRequestItem').is(':checked')) {
-      $('#storageRetrievalRequestReference input').attr('disabled', 'disabled');
-    } else {
-      $('#storageRetrievalRequestReference input').removeAttr('disabled');
-    }
-  });
-  $('#storageRetrievalRequestItem').trigger('change');
-});
-JS;
+            $(document).ready(function() {
```
Comment (EreMaijala) [inline themes/bootstrap3/templates/record/storageretrievalrequest.phtml]: Too much indentation?

**Review by EreMaijala — CHANGES_REQUESTED** [review]

File: themes/bootstrap3/templates/Recommend/TopFacets.phtml (hunk 1 above)
Comment (demiankatz) [reply themes/bootstrap3/templates/Recommend/TopFacets.phtml]: Fixed.

File: themes/bootstrap3/templates/Recommend/VisualFacets.phtml (hunk 2 above)
Comment (demiankatz) [reply themes/bootstrap3/templates/Recommend/VisualFacets.phtml]: Fixed.

File: themes/bootstrap3/templates/record/hold.phtml (hunk 3 above)
Comment (demiankatz) [reply themes/bootstrap3/templates/record/hold.phtml]: Fixed.

File: themes/bootstrap3/templates/record/illrequest.phtml (hunk 4 above)
Comment (demiankatz) [reply themes/bootstrap3/templates/record/illrequest.phtml]: Fixed.

File: themes/bootstrap3/templates/record/storageretrievalrequest.phtml (hunk 5 above)
Comment (demiankatz) [reply themes/bootstrap3/templates/record/storageretrievalrequest.phtml]: Fixed.

**Comment by demiankatz** [comment]
Thanks, @EreMaijala! I was able to adjust the templates you highlighted without the rule reapplying bad changes; I'm not sure why it did what it did in the first place.

I also took another look through templates and found a few more indentation problems that I fixed; see: 4f5878acd7fc67b7e8865580323a51308e27b68f

**Review by EreMaijala — APPROVED** [review]
Thanks @demiankatz, looks good to me now!

**Pull request merged**